day2.md

```diff
@@ -0,0 +1,171 @@
+Sensō-ji was huge: it required over an hour to just walk around the whole place. <br />
+Katherine and Matt rested in shadows while I was walking around. <br />
+<img src="day2/IMG_20160906_153415.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150819.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150810.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150831.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150924.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150959.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151518.jpg" width="500"><br/>
+Hello, Mr. Ninja!<br/>
+<img src="day2/IMG_20160906_151435.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151552.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151620.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_150731.jpg" width="500"><br/>
+A lot of shops were closed when we got there. <br/>
+<img src="day2/IMG_20160906_151812.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_151831.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_154606.jpg" width="500"><br/>
+<img src="day2/14379679_1115754091807617_7583104269522736177_o.jpg" width="500"><br/>
+
+It's time for us to go to <a name="shinjuku">Shinjuku</a> for dinner!<br />
+<img src="day2/14352325_1115752741807752_8991592585555306596_o.jpg" width="500"><br/>
+A women in pretty Kimono in front of a flower shop in the station.
+
+<img src="day2/IMG_20160906_171827.jpg" width="500"><br/>
+<img src="day2/14361400_1115754335140926_2182748393200978861_o.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_172802.jpg" width="500"><br/>
+We first went to Nakajima(中嶋), a [one star michelin](http://gm.gnavi.co.jp/shop/0120162810/) restaurant, without reservation to see if we were lucky enough to get three spots tonight or in a week. A according to the lady there, it's *impossible*.
+
+We then decided to go to Fuunji Ramen (風雲児): the chef seemed to be very famous according to the amount of [posts](http://luckypeach.com/guides/the-five-ramen-shops-you-can-and-should-visit-in-tokyo/) and [videos](https://www.youtube.com/watch?v=kNN01hPRSsU) covered it in the internet. The line was long but moved rather quickly. They served pretty fast and people ate their meal fast, too, probably because of the fact that knowing there were waiting inline literally behind you. <br />
+<img src="day2/14372101_1115754551807571_5092068918132506864_o.jpg" width="500"><br/>
+Like many legit restaurants we went to in Japan, the restaurant was pretty small and we ordered through a machine where you select what you want and put in the coins and it will give you tickets to give to a staff. <br />
+<img src="day2/IMG_20160906_183553.jpg" width="500"><br/>
+Not fully understood what all the words mean and feared of missing out, I ordered the basic ramen with a bunch of toppings. Which turned out to be way too much. Pro tip: just order the basic ramen, it comes with a bit of everything already.
```

Comment:
- "Not fully **understanding**"
- "all the words **meant**"
- "**fearing** missing out"